content/swiftui/concepts/views/terms/toggle/toggle.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'Toggle'
+Description: 'A switch that can be turned on or off.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Toggle'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+## Syntax
+
+A **`Toggle`** is switch that can be used to turn something on or off, or to choose between different options and follows this syntax:
+
+```pseudo
+Toggle(_ title: StringProtocol, isOn: Binding<Bool>)
+```
+
+The usual method for creating a text-only `'Toggle'` involves passing a string as the first parameter, rather than using a trailing closure, and a Boolean property that determines the state of the toggle (on or off).
```

Comment:
```suggestion
The usual method for creating a text-only `Toggle` involves passing a string as the first parameter, rather than using a trailing closure, and a Boolean property that determines the state of the toggle (on or off).
```